models/request/optional/optional.go

```diff
@@ -0,0 +1,7 @@
+package optional
+
+type Param[T any] *T
+
+func Of[T any](v T) Param[T] {
+	return &v
+}
```

Comment:
Maybe such functionality we could move to more "general" package somewhere closer to the "root"?

Reply:
ok, but still I would like to keep it in a models package, to make it sharable,

WDYT?

Reply:
Good idea 👍

Reply:
Moved